Parallel_test_on_Kobiton_devices_by_using_Katalon/bin/Object Repository/Heroku_web/form_login.rs

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
look like all files under `bin` are generated, is it ok to remove them ?

Reply:
I asked a member in Katalon team, those files should not be removed.